### ISSUE TITLE
How to use “quill-better-table” in vue-quill？

### ISSUE THREAD

**Issue by lnyie** [vueup/vue-quill#331]
I see that vue-quill is based on "quill": "^1.3.7", but “quill-better-table” needs to depend on “quill 2.0.x-dev.x”. Information source: https://github.com/soccerloway/quill-better-table/issues/42   Is there any support plan in the future?

**Comment by anburocky3** [comment]
Yes. Same issue. Even https://github.com/volser/quill-table-ui also depends upon 2.0 dev.

@vueup any way to support it in future?

**Comment by MmKargar** [comment]
Any one find a solution?
